habitica.el

```diff
@@ -169,6 +169,11 @@
       (define-key habitica-mode-map key habitica-command-map))
     (set-default variable key)))
 
+(defcustom habitica-show-completed-todo t
```

Comment:
I'm on the fence about having it default to true. But I think it makes sense for now to show the new feature to existing user. Could you though also make a change to the README under the customization section? The rest looks good. 
Thanks!

Reply:
Yeah sure, I thought of setting it to nil by default but then the feature might go undiscovered. I can be fine both ways though. And yes I'll update the readme and update

Reply:
By further thinking about it, it is more logically to make it disabled by default, as it is also the default behavior of API. So I have set the default value to nil and added docs.

Reply:
Ok. No problem. And if we change our mind, we can always update it later. Thank you for your contribution